docker-storage-setup.sh

```diff
@@ -720,6 +719,21 @@ unit: sectors
 
 ${dev}1 : start=     2048, size=  ${size}, Id=8e
 EOF
+}
+
+create_partition_parted(){
+local dev="$1"
+parted $dev --script mklabel gpt mkpart primary 0% 100%
```

Comment:
sfdisk creates dos type partition table by default. I think we should do same for parted. At some point of time we want to switch to GPT but this is not the patch to do so. 

We might experience surprises when we move to GPT. So use `mklabel dos`